tests/kuttl/common/assert-sample-deployment.yaml

```diff
@@ -0,0 +1,140 @@
+#
+# Check for:
+#
+# ironicAPIReadyCount: 1
+# ironicConductorReadyCount: 1
+# apiEndpoints:
+#     ironic:
+#       admin: http://ironic-admin-openstack.apps-crc.testing
+#       internal: http://ironic-internal-openstack.apps-crc.testing
+#       public: http://ironic-public-openstack.apps-crc.testing
+# we should see and validate reason and status as below on all the DBReady, DBSyncReady, BootstrapReady, DeploymentReady, ExposeServiceReady, InputReady, IronicAPIReady, IronicConductorReady, IronicRabbitMqTransportURLReady, ServiceConfigReady etc.
+# reason: Ready
+#  status: "True"
+
+apiVersion: ironic.openstack.org/v1beta1
+kind: Ironic
+metadata:
+  annotations:
+    kubectl.kubernetes.io/last-applied-configuration: |
```

Comment:
Most of the information in this field is checked afterwards, I think it's safe to remove it